components/brave_wallet/browser/fil_requests.h

```diff
@@ -13,6 +13,8 @@ namespace brave_wallet {
 
 // Returns the balance of the account of given address.
 std::string fil_getBalance(const std::string& address);
+// Returns the transactions count of given address.
+std::string fil_getTransactionCount(const std::string& address);
```

Comment:
Please put these functions inside fil namespace to match solana_request and eth_request, you can drop fil_ in the function name here too.

Reply:
done

Reply:
I believe you need to update unit test too.

Reply:
sure